src/components/WorkersList/WorkersList.tsx

```diff
@@ -0,0 +1,73 @@
+import { WorkerItem } from "../WorkerItem/WorkerItem";
+import { Worker } from "../../moudels/Worker";
+import { ReactNode, useState } from "react";
+
+export const WorkersList = () => {
+  const [workerItems, setWorkerItems] = useState<ReactNode[]>([]);
+  const fetchWorkersList = async () => {
+    try {
+      const res = await fetch("http://localhost:3030/api/employees", {
+        method: "get",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      });
+      return await res.json();
+    } catch (error) {
+      alert("Can't get workers from server!");
+      return [];
+    }
+  };
+
+  const fetchLocalTime = async (worker: Worker): Promise<Worker> => {
+    try {
+      const latLonRes = await fetch(
+        "https://api.api-ninjas.com/v1/city?name=" + worker.city,
+        {
+          method: "get",
+          headers: {
+            "Content-Type": "application/json",
+            "X-Api-Key": "l3GnNztJXP8ba/EKYgkkVw==03kE5vXWxMyvZvBy",
+          },
+        }
+      );
+      const latLonAns = await latLonRes.json();
+      const timeRes = await fetch(
+        "https://api.api-ninjas.com/v1/worldtime?lat=" +
+          latLonAns[0].latitude +
```

Comment:
would have liked latLonAns to not be array and then we have to get the zeroth location